### ISSUE TITLE
侧边栏post链接跳转异常

### ISSUE THREAD

**Issue by chenjiageng** [iissnan/hexo-theme-next#1836]
### I agree and want to create new issue 

- [x] Yes, I was on [Hexo Docs page](https://hexo.io/docs/), especially on [Templates](https://hexo.io/docs/templates.html), [Variables](https://hexo.io/docs/variables.html), [Helpers](https://hexo.io/docs/helpers.html) and [Troubleshooting](https://hexo.io/docs/troubleshooting.html).
- [x] Yes, I was on [NexT Documentation Site](http://theme-next.iissnan.com/).
- [x] And yes, I already searched for current [issues](https://github.com/iissnan/hexo-theme-next/issues?utf8=%E2%9C%93&q=is%3Aissue) and this is not help to me.

***

### Expected behavior 
侧边栏 post 点击后跳转至 archives 

### Actual behavior 
侧边栏日志点击后跳转至/archives/%20%7C%7C%20archive
但categories 和tags 跳转正常

### Steps to reproduce the behavior 
1. 通过 git clone https://github.com/iissnan/hexo-theme-next themes/next 下载
2. 通过http://theme-next.iissnan.com/getting-started.html 帮助填写站点配置
3. 在本地部署和上传至github后 点击 侧边栏中的 post
4.删除name.github.io文件夹，重新安装和配置， 问题重现！！！
* Link to demo site with this issue: https://chenjiageng.github.io/
* Link(s) to source code or any usefull link(s): https://github.com/chenjiageng

### NexT Information

**NexT Version:**

```
[x] Master
[] Latest Release
[] Old version - 
```

**NexT Scheme:**

```
[x] All schemes
[] Muse
[] Mist
[] Pisces
[] Gemini
```

**Comment by ivan-nginx** [comment]
Fixed in ba842f5.

**Comment by chamwen** [comment]
In your config file of NEXT theme

```
archives: /archives/ || archive
# change to
archives: /archives
```

Reference:
[常见BUGs](https://www.chamwen.com/2018/03/08/tool_hexo/)
